gpbft/gpbft.go

```diff
@@ -796,7 +796,7 @@ func (i *instance) broadcast(round uint64, step Phase, value ECChain, ticket Tic
 	}
 
 	gmsg := &GMessage{
-		Sender:        i.participant.id,
+		Sender:        i.participant.host.ID(),
```

Comment:
Same as above. I believe we want to remove `GMessage.Sender` and let `RequestBroadcast` sign, etc. The host may need to broadcast _multiple_ messages (one per miner).